sdk-extensions/autoconfigure/README.md

```diff
@@ -122,10 +122,10 @@ The [OpenTelemetry Protocol (OTLP)](https://github.com/open-telemetry/openteleme
-| otel.exporter.otlp.protocol                              | OTEL_EXPORTER_OTLP_PROTOCOL                              | The transport protocol to use on OTLP trace, metric, and log requests. Options include `grpc` and `http/protobuf`. Default is `grpc`.                                                                                                                                                                                                                                                                                                                         |
-| otel.exporter.otlp.traces.protocol                       | OTEL_EXPORTER_OTLP_TRACES_PROTOCOL                       | The transport protocol to use on OTLP trace requests. Options include `grpc` and `http/protobuf`. Default is `grpc`.                                                                                                                                                                                                                                                                                                                                          |
-| otel.exporter.otlp.metrics.protocol                      | OTEL_EXPORTER_OTLP_METRICS_PROTOCOL                      | The transport protocol to use on OTLP metric requests. Options include `grpc` and `http/protobuf`. Default is `grpc`.                                                                                                                                                                                                                                                                                                                                         |
-| otel.exporter.otlp.logs.protocol                         | OTEL_EXPORTER_OTLP_LOGS_PROTOCOL                         | The transport protocol to use on OTLP log requests. Options include `grpc` and `http/protobuf`. Default is `grpc`.                                                                                                                                                                                                                                                                                                                                            |
+| otel.exporter.otlp.protocol                              | OTEL_EXPORTER_OTLP_PROTOCOL                              | The transport protocol to use on OTLP trace, metric, and log requests. Options include `grpc` and `http/protobuf`. Versions 2.0+ of the Java agent use `http/protobuf` as the default protocol, not `grpc`.                                                                                                                                                                                                                                                                                                                         |
```

Comment:
In this project the default protocol is `grpc`. Perhaps we could use a footnote for this? I'd keep the existing text and add `**[1]**` and after the table add ```**[1]**: NOTE: OpenTelemetry Java Agent 2.x uses `http/protobuf` by default.```

Reply:
Good advice, done.